spec/features/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'capybara_helper'
+
+describe 'View / Profile' do
+  describe 'set user timezone' do
+    context 'as a logged in user' do
+      let(:user) { User.find_or_create_by(name: 'test user', email: 'test@example.justice.gov.uk') }
+
+      before do
+        login_as!(user)
+        visit 'user/edit'
+      end
+
+      it 'displays my current timezone' do
+        expect(page).to have_content(I18n.t(:lede_html, scope: [:users, :edit], timezone: user.timezone))
+      end
+
+      context 'I can select a timezone' do
```

Comment:
minor semantic point - this isn't quite a `context` (there's no different condition or setup inside this block) it's more of a `describe` (in this case, you're describing what it should do). It doesn't make any functional difference of course

Reply:
Agreed